apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -202,7 +202,10 @@ async fn create_client(
         .status(status)
         .header("Access-Control-Allow-Origin", "*")
         .header("Access-Control-Allow-Method", "GET, POST")
-        .header("Access-Control-Allow-Headers", "Content-Type")
+        .header(
+            "Access-Control-Allow-Headers",
+            "Authorization, Content-Type",
+        )
```

Comment:
I guess this is a follow-up to https://github.com/514-labs/moose/pull/1326 ?

I needed to allow `Authorization` header to get my frontend to use auth with moose.